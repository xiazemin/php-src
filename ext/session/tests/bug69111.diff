001+ Warning: SessionHandler::open(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 10
002+ 
003+ Warning: SessionHandler::write(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 11
004+ 
005+ Warning: SessionHandler::read(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 12