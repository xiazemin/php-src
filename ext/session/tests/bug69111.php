<?php
$sh = new SessionHandler;
session_set_save_handler($sh);

$savePath = ini_get('session.save_path');
$sessionName = ini_get('session.name');

// session_start(); // Uncommenting this makes it not crash when reading the session (see below), but it will not return any data. 

$sh->open($savePath, $sessionName);
$sh->write("foo", "bar");
var_dump($sh->read(@$id));
?>
