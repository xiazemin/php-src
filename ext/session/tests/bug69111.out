Warning: SessionHandler::open(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 10

Warning: SessionHandler::write(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 11

Warning: SessionHandler::read(): Session is not active in /Users/didi/PhpstormProjects/c/php-src/ext/session/tests/bug69111.php on line 12
bool(false)